### PR TITLE
Orchestrating third party services with oauth2 updates

### PR DESCRIPTION
**Pull request by wmedvede**
- Updates the Prerequisites for the guide

**JIRA:**

**Description:**

Please make sure that your PR meets the following requirements:

- [X] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [X] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [X] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [X] The nav.adoc file has a link to this guide in the proper category
- [X] The index.adoc file has a card to this guide in the proper category, with a meaningful description

<details>
<summary>
How to setup automatic cherry-pick to another branch ?
</summary>

The cherry-pick action allows to setup automatic cherry-pick from `main` to a specific branch.

To allow it, you will need to add the corresponding label with pattern: `backport-{RELEASE_BRANCH}`.  
For example, if a backport to branch `1.26.x` is needed, then the label should be `backport-1.26.x`.

Once the PR is merged, the action will retrieve the commit and cherry-pick it to the desired branch.

*NOTE: You can still add label after the merge and it should still be cherry-picked.*
</details>

### CODE REVIEW

**Review by kaldesai — APPROVED** [review]
@wmedvede Your content looks good to me. Thank you!

File: serverlessworkflow/modules/ROOT/pages/security/orchestrating-third-party-services-with-oauth2.adoc [hunk 1]
```diff
@@ -456,8 +456,11 @@ Also, you can use the alternatives defined in the link:{quarkus_guide_config_ref
 Once you clone the `serverless-workflow-oauth2-orchestration-quarkus` example application from GitHub repository, you can run the example application.
 
 .Prerequisites
-* You have an account in GitHub.
-* Apache Maven {maven_min_version} or later is installed.
+* link:{java_install_url}[Java] {java_min_version} is installed.
+* link:{maven_install_url}[Maven] {maven_min_version} or later is installed.
+* link:{docker_install_url}[Docker] {docker_min_version} or later is installed.
+* (Optional) link:{docker_compose_install_url}[Docker compose] {docker_compose_min_version} or later is installed.
```
Comment (radtriste) [inline serverlessworkflow/modules/ROOT/pages/security/orchestrating-third-party-services-with-oauth2.adoc:462]: nitpicking but is there a way to use podman instead ?

Comment (wmedvede) [reply serverlessworkflow/modules/ROOT/pages/security/orchestrating-third-party-services-with-oauth2.adoc:462]: Not sure it's a big win here. In general around the other examples, and guides, that use for example minikube the container platform used is Docker.

Comment (radtriste) [reply serverlessworkflow/modules/ROOT/pages/security/orchestrating-third-party-services-with-oauth2.adoc:462]: fair enough

**Pull request merged**